java/ql/lib/semmle/code/java/frameworks/android/Widget.qll

```diff
@@ -5,7 +5,7 @@ private import semmle.code.java.dataflow.ExternalFlow
 private import semmle.code.java.dataflow.FlowSources
 
 private class DefaultAndroidWidgetSources extends RemoteFlowSource {
```

Comment:
This class can be completely removed now, sources with `remote` kind are automatically picked up.

Reply:
Should I also remove this [RemoteFlowSources module](https://github.com/github/codeql/blob/d27f84e34e820bcbd81dc0cc22e497682bc1e55f/java/ql/lib/semmle/code/java/dataflow/FlowSources.qll#L39-L44) since the only import it has is `semmle.code.java.frameworks.android.Widget`?

Reply:
Yes, good point.